src/payment/checkout/payment-form/PaymentForm.jsx

```diff
@@ -137,6 +141,20 @@ export class PaymentFormComponent extends React.Component {
     return errors;
   }
 
+  validateAsciiNames(firstName, lastName) {
+    const errors = {};
+
+    if (
+      firstName
+      && lastName
+      && !/[A-Za-z]/.test(firstName + lastName)
```

Comment:
Do we need to allow spaces, apostrophes, and/or hyphens too?

Reply:
It just requires that there are _any_ ascii letters, but it's ok if there are other characters as well.